sklearn/utils/_joblib.py

```diff
@@ -2,13 +2,16 @@
 # site one
 from __future__ import absolute_import
 import os as _os
+import warnings as _warnings
 
 # An environment variable to use the site joblib
 if _os.environ.get('SKLEARN_SITE_JOBLIB', False):
-    from joblib import __all__
-    from joblib import *  # noqa
-    from joblib import __version__
-    from joblib import logger
+    with _warnings.catch_warnings():
+        _warnings.simplefilter("ignore")
```

Comment:
Why is this needed? (does joblib raise warnings on import?) Maybe adding a short comment would be useful.